react.gradle

```diff
@@ -340,16 +340,14 @@ afterEvaluate {
                     include "**/libhermes*.so"
                 }
             }.visit { details ->
-                def targetVariant = ".*/transforms/[^/]*/${targetPath}/.*"
-                def path = details.file.getAbsolutePath().replace(File.separatorChar, '/' as char)
-                if (path.matches(targetVariant) && details.file.isFile()) {
+                if (details.file.isFile()) {
```

Comment:
Checking whether this is a file should be enough, as we are only visiting a single directory